src/Microsoft.DotNet.Interactive.ApiCompatibility.Tests/ApiCompatibilityTests.Interactive_api_is_not_changed.approved.txt

```diff
@@ -424,9 +425,9 @@ Microsoft.DotNet.Interactive.Commands
   public class RequestValueInfos : KernelCommand
     .ctor(System.String targetKernelName = null)
   public class SendEditableCode : KernelCommand
-    .ctor(System.String language, System.String code, System.String targetKernelName = vscode)
+    .ctor(System.String kernelName, System.String code, System.String targetKernelName = vscode)
```

Comment:
What's the difference here between `kernelName` and `targetKernelName`?

Also, I think having a default of `vscode` at this layer is wrong.

Reply:
You're right, `kernelName` can (should?) go.  I'm thinking `targetKernelName` shouldn't have a default; it _must_ be specified.

Reply:
Looking back there is a distinction to be made between `kernelName` and `targetKernelName`, but the naming should certainly be improved.

`targetKernelName` is the kernel that will handle adding the new code cell, and while "vscode" isn't the right place for that value here, that's essentially what it will be in practice.

`kernelName` is the name of the kernel that will be set on the new cell that will ultimately receive the `SendValue` command.